Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -38,6 +38,8 @@ public sealed class SandboxedProcessPipExecutor : ISandboxedProcessFileStorage
         /// </summary>
         public const int MaxConsoleLength = 2048; // around 30 lines
 
+        private const uint DeadExitCode = 0xDEAD;

```

Comment:
Maybe a comment that this is specific to cloudbuild only?

Reply:
I like Danny's suggestion to name this AzureWatsonExitCode. When AzureWatson isn't active, this isn't applicable